src/script/lua_api/l_mainmenu.cpp

```diff
@@ -993,7 +994,8 @@ int ModApiMainMenu::l_get_screen_info(lua_State *L)
 int ModApiMainMenu::l_get_package_list(lua_State *L)
 {
 	std::string url = g_settings->get("contentdb_url");
-	std::vector<Package> packages = getPackagesFromURL(url + "/api/packages/");
+	std::vector<Package> packages = getPackagesFromURL(url +
+			"/api/packages/?type=mod&type=game&type=txp&protocol_version=" LATEST_PROTOCOL_VERSION_STRING);
```

Comment:
Line too long. Looks good otherwise.

Reply:
Is a `+` missing before `LATEST_PROTOCOL_VERSION_STRING`?

Reply:
No. That's a C syntax on concatenating strings